### PR TITLE
Request my site as featured Gatsby site

### PR DESCRIPTION
**Pull request by jarodpeachey**
This is requesting my site to be featured as a Gatsby site.

### CODE REVIEW

**Comment by laurieontech** [comment]
Thanks for the PR! Right now the tests are failing because CMS:Netlify is not an included category. If you remove that we can go ahead and review this for merging.

**Comment by laurieontech** [comment]
Unfortunately this change won't pass the test as Netlify is not a category.

The valid categories are: 
API, Accessibility, Agency, App, Blog, Business, Community, Conference, Consulting, Data, Design, Design System, Directory, Documentation, Education, Entertainment, Entrepreneurship, Event, Featured, Finance, Food, Freelance, Gallery, Government, Healthcare, Human Resources, JavaScript, Law, Landing Page, Learning, Library, Marketing, Media, Mobile Development, Music, Nonprofit, Open Source, Photography, Podcast, Portfolio, Productivity, Programming, Real Estate, Retail, SEO, Science, Security, Sports, Technology, Travel, User Experience, Video, Web Development, WordPress, eCommerce

I'm happy to make the change and merge it, but wanted to give you a chance to select any other categories you might find relevant!

**Comment by laurieontech** [comment]
Looks great! Thanks for hanging with me on those changes :)

File: docs/sites.yml [hunk 1]
```diff
@@ -8816,3 +8816,18 @@
   built_by: Amrish Kushwaha
   built_by_url: https://github.com/isamrish
   featured: false
+- title: Jarod Peachey
+  main_url: https://jarodpeachey.netlify.com
+  url: https://jarodpeachey.netlify.com
+  # optional: for open-source sites, this URL points to the repo that powers the site
+  source_url: https://github.com/jarodpeachey/portfolio-website
```
Comment (muescha) [inline docs/sites.yml]: it is: https://github.com/jarodpeachey/portfolio

**Comment by muescha** [comment]
@jarodpeachey your linkedin link goes to a spammy https://linkeding.com/in/jarod-peachey -> you should remove the additional `g`: linkedin**g** .com

**Pull request merged**